src/data/labelbox/labelbox_wrapper.py

```diff
@@ -0,0 +1,10 @@
+from pathlib import Path
+
+class LabelBoxWrapper:
```

Comment:
labelbox was postponed since they require login with company email. but it could be extended in the future